ts/src/test/Exchange/base/test.sharedMethods.ts

```diff
@@ -537,6 +537,13 @@ function assertOrderState (exchange, skippedProperties, method, order, assertedS
     }
 }
 
+function getNonInactiveMarkets (exchange) {
```

Comment:
i coulnd't find other name, eg neither `getActiveMarkets` nor `getActiveOrUndefinedMarkets` fits well

Reply:
`getActiveMarkets` is better that non inactive

Reply:
eg. it's like having `good, bad, unknown`, we want "non-bad" ones.

Reply:
so, i wanted to mean "not inactive markets" per se, but ok, i'll rename not a big deal